### PR TITLE
K8s-2844: Add cert-manager addon docs

### PR DESCRIPTION
**Pull request by cbeneke**

### CODE REVIEW

**Comment by bashofmann** [comment]
The tutorial https://docs.syseleven.de/metakube/de/tutorials/create-an-ingress-controller should be referenced and adapted.

**Comment by cbeneke** [comment]
This is currently WIP, sorry forgot to mark so :) But thanks for the hint :)

File: pages/04.tutorials/15.create-an-ingress-controller/default.en.md [hunk 1]
```diff
@@ -50,9 +56,11 @@ helm repo update
 helm install --name cert-manager --namespace cert-manager --version v0.7.0 jetstack/cert-manager
 ```
 
+If you want to use [our DNS](https://docs.syseleven.de/syseleven-stack/en/reference/dns) for certificate DNS validation (e.g. required for wildcard certificates) you need to install the [designate webhook](https://github.com/syseleven/designate-certmanager-webhook) in the cluster. Please follow instructions from the provided [README](https://github.com/syseleven/designate-certmanager-webhook/blob/master/README.md).
```
Comment (shahbazn) [inline pages/04.tutorials/15.create-an-ingress-controller/default.en.md]: nit-pick: I would change `our DNS` to `SysEleven DNS service`

Comment (cbeneke) [reply pages/04.tutorials/15.create-an-ingress-controller/default.en.md]: Fixed in amend commit

**Pull request merged**